solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/get.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import { PaabHelper } from '../../../support/complex/paabHelper';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('get environment type configs', () => {
+  const setup: Setup = Setup.getSetup();
+  const envTypeId = setup.getSettings().get('envTypeId');
+  const envTypeConfigId = setup.getSettings().get('envTypeConfigId');
+  const paabHelper: PaabHelper = new PaabHelper();
+  let itAdminSession: ClientSession;
+  let pa1Session: ClientSession;
+  let researcherSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    const paabResources = await paabHelper.createResources();
+    itAdminSession = paabResources.adminSession;
+    pa1Session = paabResources.pa1Session;
+    researcherSession = paabResources.rs1Session;
+  });
+
+  afterAll(async () => {
+    await paabHelper.cleanup();
+  });
+
+  describe('get environments type configs (non-project route)', () => {
+    test('succeeds as an ITAdmin', async () => {
+      const { data: response } = await itAdminSession.resources.environmentTypes
+        .environmentType(envTypeId)
+        .configurations()
+        .environmentTypeConfig(envTypeConfigId)
+        .get();
+
+      expect(response.id).toBe(envTypeConfigId);
+    });
+
+    test('fails as a ProjectAdmin', async () => {
+      try {
+        await pa1Session.resources.environmentTypes
+          .environmentType(envTypeId)
+          .configurations()
+          .environmentTypeConfig(envTypeConfigId)
+          .get();
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
+            error: 'User is not authorized'
+          })
+        );
+      }
+    });
+
+    test('fails as a Researcher', async () => {
+      try {
+        await researcherSession.resources.environmentTypes
+          .environmentType(envTypeId)
+          .configurations()
+          .environmentTypeConfig(envTypeConfigId)
+          .get();
```

Comment:
Do we want to throw an error after the `await` statement to guarantee that the `await` statement threw an error? 

An example is shown [here](https://github.com/aws-solutions/research-service-workbench-on-aws/blob/d035cc34c555b521cc50e5aab88924e5c39c2fa1/solutions/swb-reference/integration-tests/tests/multiStep/costCenter.test.ts#L38). We might have a false negative where the test passes because the `await` statement doesn't throw an error.

Reply:
Actually I don't think this is needed for this scenario since you already have `jest.hasAssertions` and each of these tests only have one assertion.